app/modules/submissions/models.py

```diff
@@ -240,6 +240,8 @@ def git_clone(self, project):
         return repo
 
     @classmethod
+    # i believe if called *without* transaction_id this would be kinda nonsensical cuz it would call import_tus_files()
```

Comment:
Seems sensible to me. In which case, remove the default and assert that it isn't

Reply:
done.